logs/fluent-bit/k8s-manifest/fluentbit-ds.yaml

```diff
@@ -34,6 +34,8 @@ spec:
                 name: coralogix-keys
             - configMapRef:
                 name: fluent-bit
+            - configMapRef:
+                name: fluent-bit-env
```

Comment:
Who is applying this configMap?

Reply:
The README.md states they have to manually create one by that name.